R/derive_params_growth_height.R

```diff
@@ -328,11 +326,11 @@ derive_params_growth_height <- function(dataset,
       mutate(!!!set_values_to_pctl)
 
     dataset_final <- bind_rows(dataset_final, add_pctl, unmatched_pctl) %>%
-      select(-c(L, M, S, sex_join, heightu_join, meta_height, ht_diff, is_lowest))
+      select(-c(L, M, S, sex_join, heightu_join, meta_height))
   }
 
   dataset_final <- dataset_final %>%
-    select(-c(SD2pos, SD3pos, SD2neg, SD3neg))
+    select(-c(SD2pos, SD3pos, SD2neg, SD3neg, ht_bins))
 
   return(dataset_final)
 }
```

Comment:
Could you please double check where `temp_z` and `temp_val` should be dropped ? they are still in the final dataset. Thanks